ryoanji/test/nbody/traversal_cpu.cpp

```diff
@@ -102,7 +102,7 @@ TEST(Gravity, TreeWalk)
     auto   t0       = std::chrono::high_resolution_clock::now();
     double egravTot = computeGravity(octree.childOffsets.data(), octree.internalToLeaf.data(), centers.data(),
                                      multipoles.data(), layout.data(), 0, octree.numLeafNodes, x, y, z, h.data(),
-                                     masses.data(), G, ax.data(), ay.data(), az.data());
+                                     masses.data(), box, G, ax.data(), ay.data(), az.data());
```

Comment:
It is strange that the return value of computeGravity() is the potential. Why should that be considered the final result of the computation? Especially when the other "out" values (ax,ay,az) are in the arguments list, I would make computeGravity() void and move egravTot to the arguments.